packages/selection/src/useControlled.js

```diff
@@ -0,0 +1,21 @@
+/**
```

Comment:
Having a hook to mimic the `ControlledComponent` utility from before might not be necessary in the hooks world.

When I originally created it the use-cases seemed simple, but the whole `onStateChange` prop always got confusing as the components grew in complexity.

Since hooks normally treat their arguments as the "props" maybe we could have a more simplified flow by not including it immediately.

Reply:
yeah I think you're right, was trying to see how this works but I don't think we need it.